lib/Steps/Integrations/NextJs.ts

```diff
@@ -11,7 +11,7 @@ import { debug, green, l, nl, red } from '../../Helper/Logging';
 import { SentryCli, SentryCliProps } from '../../Helper/SentryCli';
 import { BaseIntegration } from './BaseIntegration';
 
-const COMPATIBLE_NEXTJS_VERSIONS = '>=10.0.8 <13.0.0';
+const COMPATIBLE_NEXTJS_VERSIONS = '>=10.0.8 <=13.0.0';
```

Comment:
```suggestion
const COMPATIBLE_NEXTJS_VERSIONS = '>=10.0.8 <14.0.0';
```

Reply:
Oh, damn. You're totally right! LOL 

Thanks for catching that!